presidio-analyzer/analyzer/entity_recognizer.py

```diff
@@ -198,12 +208,13 @@ def __add_n_words(index,
                             take the successing words
         """
         i = index
-        # collect at most n words
+        # collect at most n words (in lower case)
         remaining = n_words
         while 0 <= i < len(lemmas) and remaining > 0:
-            if lemmas[i] in lemmatized_filtered_keywords:
+            lower_lemma = lemmas[i].lower()
```

Comment:
I am not sure `toLower` is the best practice.
[https://github.com/explosion/spaCy/issues/2420](https://github.com/explosion/spaCy/issues/2420)
[https://spacy.io/api/token#attributes](https://spacy.io/api/token#attributes) (search `lower`)

Reply:
Take a look at this: https://github.com/explosion/spaCy/issues/1901
When we don't use a tagger (which is disabled in our case), spacy does a lookup table based lemmatization. When using a tagger, the lemmas are always lowercase. This should be a better solution.

Reply:
Good points, however @itye-msft, I am interested in lemmas and not tokens, so this solution doesnt apply.
@omri374 not sure why, but disabling the tagger didn't work